docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -256,6 +260,37 @@ difference is that you would specify your Hibernate ORM configuration in `META-I
 </persistence>
 --
 
+== Defining entities in external projects or jars
+
+Hibernate ORM in Quarkus relies on compile-time bytecode enhancements to your entities. If you define your entities in the
```

Comment:
Inconsistent use of `{project-name}` placeholder. Sometimes `{project-name}` is used and sometimes `Quarkus` is used. I suppose it should be `{project-name}`.

Reply:
no, we don't need project-name anymore

Reply:
Cool! I will take into account when I modify documentation :)